Client/game_sa/CWorldSA.h

```diff
@@ -16,6 +16,7 @@
 #define FUNC_ProcessLineOfSight                             0x56BA00 // ##SA##
 #define FUNC_FindGroundZFor3DCoord                          0x5696C0 // ##SA##
 #define FUNC_FindGroundZForCoord                            0x569660 // ##SA##
+#define FUNC_FindRoofZFor3DCoord                            0x569750 // ##SA##
 #define FUNC_CTimer_Stop                                    0x561AA0 // ##SA##
 #define FUNC_CTimer_Update                                  0x561B10 // ##SA##
```

Comment:
This is just an old bad design. There's no need to define a macro for function address, you can just hardcode it when calling the function in `*SA.cpp` file.

This is enough and readible:
```
auto CWorld_FindRoofZFor3DCoord = (float(__cdecl*)(CVector, bool*))0x0569750;
```
Anyways, the PR looks done, so let's just merge it.

Reply:
I just did it because of consistency. I know its the same as just putting it there, but it seemed like a better idea to follow the convetion.